parts/k8s/cloud-init/artifacts/mountetcd.sh

```diff
@@ -12,7 +12,7 @@ if mount | grep $MOUNTPOINT; then
   umount /dev/sdc1
 fi
 if ! grep "/dev/sdc1" /etc/fstab; then
```

Comment:
```suggestion
if ! grep "etcd_disk" /etc/fstab; then
```

Reply:
This suggestion (again, see #3259) will have no effect upon reboot scenarios, as this script only runs once (during cluster provisioning)

Reply:
why are we even doing that check then? if the script only runs once, seems like that check shouldn't be needed. What am I missing?

Reply:
It was added recently to support the script being re-run *during provisioning*. See:

https://github.com/Azure/aks-engine/blob/master/parts/k8s/cloud-init/artifacts/cse_config.sh#L24

Reply:
I would actually check for the mount point being in the fstab since that would be a variable and it does not depend on which way it may be in the fstab.

I fully agree with the "smallest change to get clusters stable" first so it likely does not matter but correctness means the fstab line and the if/grep line need to be changed.